packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -101,6 +78,21 @@ export default class MediaThemeMux extends MediaTheme {
       `,
       this.shadowRoot as Node
     );
+
+    // Possibly add this functionality to the base class MediaTheme.
+    this.shadowRoot?.querySelectorAll('*').forEach((element) => {
```

Comment:
1. good point, just to be safe because when changing permutations some elements will be replaced and need the `part` attribute again. the goal is to have the theme developer not worry about this logic that can be automated.

2. currently it's still needed but I believe once we add these `top|center|bottom` parts in Media Chrome it's possible to remove the section parts for each component and just target the part and put the right CSS var in it.

Reply:
cool makes sense to me. Something to keep an eye on for improving at some point.